src/StatusQ/Controls/StatusSeedPhraseInput.qml

```diff
@@ -40,24 +40,58 @@ import StatusQ.Core.Theme 0.1
        }
    \endqml
 
+   \image status_seed_phrase_input.png
+
    For a list of components available see StatusQ.
 */
 
 Item {
     id: root
     width: 162
     height: 44
+    /*!
+        \qmlproperty textEdit
```

Comment:
General for all properties: If you use the syntax `StatusSeedPhraseInput::nameOfProperty` the docu makes that text clickable and navigable. Same if you have a method or signal.
It is also common to add the property type: `\qmlproperty propertyType Type::nameOfProperty`

Reply:
done